.npmignore

```diff
@@ -1,4 +1,11 @@
+# Be aware that defininig patterns in "files" on package.json
```

Comment:
```suggestion
# Be aware that defining patterns in "files" on package.json
```

Reply:
Removed `.npmignore` in favor of `"files"`.